feature/common/src/main/res/values/colors.xml

```diff
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="purple_200">#FFBB86FC</color>
-    <color name="purple_500">#FF6200EE</color>
-    <color name="purple_700">#FF3700B3</color>
-    <color name="teal_200">#FF03DAC5</color>
-    <color name="teal_700">#FF018786</color>
-    <color name="black">#FF000000</color>
-    <color name="white">#FFFFFFFF</color>
+    <color name="orange">#F2742E</color>
+    <color name="orange_20">#33F2742E</color>
```

Comment:
Where is this color used? (orange_20)

Reply:
This color is used for BottomNavgationBar item was selected.